api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionSignPaths.scala

```diff
@@ -85,8 +93,58 @@ trait SubmissionSignPaths
 
     onComplete(fSubmission) {
       case Success(sub) =>
+        if (sub.status.code == 10) {
```

Comment:
Can this be replaced with `sub.status == Signed`? It would read clearer and be more type safe.